[2025-03-24T12:34:15.735+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:34:15.841+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2025-03-24T12:34:15.892+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2025-03-24T12:34:15.897+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:34:16.028+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): download_source_data> on 2021-02-02 06:00:00+00:00
[2025-03-24T12:34:16.077+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'download_source_data', 'scheduled__2021-02-02T06:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpjp1q9zmi']
[2025-03-24T12:34:16.079+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask download_source_data
[2025-03-24T12:34:16.059+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2846) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:34:16.082+0000] {standard_task_runner.py:72} INFO - Started process 2856 to run task
[2025-03-24T12:34:16.448+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2021-02-02T06:00:00+00:00 [running]> on host 4791d3610749
[2025-03-24T12:34:17.071+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='download_source_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-02T06:00:00+00:00'
[2025-03-24T12:34:17.072+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:34:17.203+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:34:17.204+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2021-02']
[2025-03-24T12:34:17.306+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:34:17.309+0000] {subprocess.py:106} INFO - 2021-02
[2025-03-24T12:34:17.327+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:34:17.442+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:34:17.452+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=download_source_data, run_id=scheduled__2021-02-02T06:00:00+00:00, execution_date=20210202T060000, start_date=20250324T123415, end_date=20250324T123417
[2025-03-24T12:34:17.689+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:34:17.914+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:34:17.924+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
